ibl_to_nwb/NwbToOne/nwb_to_ibl_metadata.py

```diff
@@ -1,14 +1,29 @@
-from pynwb import NWBHDF5IO
+import json
+from oneibl.one import ONE
+from pynwb import NWBFile, NWBHDF5IO
 import uuid
+import h5py
 import sys
 from datetime import datetime
-from .field_map import *
+from .field_map import field_map_nwbfile, field_map_session_data, \
+    field_map_subject,field_map_IBL_subject
 from ndx_ibl_metadata import IblProbes
 from copy import copy, deepcopy
 import h5py
 
 
 def _convert_numpy_to_python_dtype(dict_to_convert_in):
+    """
+    Converts numpy dtypes like np.int*, np.float* to python native int, float
+    Conversion useful to store data in .json format which supports only native
+    datatypes.
+    Parameters
+    ----------
+    dict_to_convert_in: dict
```

Comment:
Can you specify what are supposed to be the keys and values of this dictionary?

Reply:
Some numeric fields in nwbfile are stored as a np type( np.int, np.float), but this cannot be serialized in a json format. 
Writing to json requires all numeric values to be of native formats: np.float as just float, np.int as just int. This function makes these conversions by looping over a dicts values and checking for such types
Example:
a=np.int16(10)
type(a)
# numpy.int16
type(a.item())
# int (python native format) This can be converted to a valid json type using json.dump()

Reply:
https://pynative.com/python-serialize-numpy-ndarray-into-json/